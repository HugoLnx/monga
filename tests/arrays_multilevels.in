void main() {
	int[][] array;
	array = new int[][2];
	array[0] = new int[2];
	array[1] = new int[3];
  array[0][0] = 10;
  array[0][1] = 15;
  array[1][0] = 20;
  array[1][1] = 25;
  array[1][2] = 30;
	printf("array[0][0]: %d\n", array[0][0]);
	printf("array[0][1]: %d\n", array[0][1]);
	printf("array[1][0]: %d\n", array[1][0]);
	printf("array[1][1]: %d\n", array[1][1]);
	printf("array[1][2]: %d\n", array[1][2]);
}
