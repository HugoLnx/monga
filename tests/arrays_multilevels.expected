array[0][0]: 310
array[0][1]: 315
array[1][0]: 320
array[1][1]: 325
array[1][2]: 330
